### PR TITLE
feat(atomic,headless): add clearFilters option on search box

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-1505

### CODE REVIEW

**Review by jpmarceau — APPROVED** [review]
👌

**Review by spiracha-coveo — APPROVED** [review]
LGTM

**Review by btaillon — APPROVED** [review]

File: packages/headless/src/features/search/search-actions.ts [hunk 1]
```diff
@@ -105,6 +105,14 @@ export interface ExecuteSearchThunkReturn {
   analyticsAction: SearchAction;
 }
 
+export interface PrepareForSearchWithQueryOptions {
```
Comment (olamothe) [inline packages/headless/src/features/search/search-actions.ts:108]: Unless I'm mistaken, this does not need to be exported ? Or does it affect the documentation ?

**Review by olamothe — APPROVED** [review]

File: packages/headless/src/features/search/search-actions.ts (hunk 1 above)
Comment (ThibodeauJF) [reply packages/headless/src/features/search/search-actions.ts:108]: Let's try
```suggestion
interface PrepareForSearchWithQueryOptions {
```

**Pull request merged**